source/guides/guides/continuous-integration.md

```diff
@@ -700,6 +700,23 @@ See {% url bahmutov/yarn-cypress-cache https://github.com/bahmutov/yarn-cypress-
 
 If you are running long runs on Docker, you need to set the `ipc` to `host` mode. {% issue 350 'This issue' %} describes exactly what to do.
 
+In a Docker container, the default size of the `/dev/shm` shared memory space is 64MB. This is not typically enough to run Chrome and can cause the browser to crash. You can fix this by passing the `--disable-dev-shm-usage` flag to Chrome with the following workarounds:
+
+
+```javascript
+module.exports = (on, config) => {
+  on('before:browser:launch', (browser = {}, launchOptions) => {
+    if (browser.family === 'chrome') {
```

Comment:
So I didn't run this code before, but running this is actually inaccurate. It should be:
```suggestion
    if (browser.family === 'chromium' && browser.name !== 'electron') {
```